templates/style.css

```
@@ -520,4 +516,12 @@ body.darkmode .btn-info {
 body.darkmode .panel_toolbox > li > a:hover {
     background: transparent;
 }
+
+body.darkmode div pre {
+    background-color: #3e3939 !important;
+    color: #eee;
+}
+
+body.darkmode span.tag {
+    color: yellow;
```

Changing this to a less hard color of yellow so it will be more in line with the default used colors
```suggestion
    color: #FFD119;
```